strikt-jvm/src/main/kotlin/strikt/java/time/TemporalAccessor.kt

```diff
@@ -30,12 +32,13 @@ infix fun <T : TemporalAccessor> Assertion.Builder<T>.isBefore(expected: Tempora
       is Year -> it.isBefore(Year.from(expected))
       is YearMonth -> it.isBefore(YearMonth.from(expected))
       is ZonedDateTime -> it.isBefore(ZonedDateTime.from(expected))
+      is OffsetDateTime -> it.isBefore(OffsetDateTime.from(expected))
```

Comment:
My use case requires support for [OffsetDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/OffsetDateTime.html), so I added OffsetDateTime to isBefore and isAfter as well.